code/obj/nuclearreactor/nuclearreactor.dm

```diff
@@ -32,6 +32,8 @@
 	var/radiationLevel = 0
 	/// Current gas mixture to process
 	var/datum/gas_mixture/current_gas = null
```

Comment:
is current_gas still used at all?

Reply:
Yes, it's basically the gas currently stored in the reactor casing. `air_contents` is basically `current_gas` after it's been processed.